src/gui/ProjectNotes.cpp

```diff
@@ -26,6 +26,7 @@
 #include "ProjectNotes.h"
 
 #include <QAction>
+#include <QActionGroup>
```

Comment:
@michaelgregorius is there any purpose for this specific header?

Reply:
It did not compile without it.

Reply:
ok